CameraBrowser/CameraBrowserSample/CameraBrowserExtension.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using ImageCaptureCore;
+using System.Linq;
+
+namespace CameraBrowserSample
+{
+	public static class CameraBrowserExtension 
+	{
+		public static bool canSyncClock(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanSyncClock);
```

Comment:
We use [PascalCase](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers) in our source code:

```suggestion
		public static bool CanSyncClock(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanSyncClock);
```